### PR TITLE
[en] add pics to middleware-minio-create

### PR DESCRIPTION
**Pull request by ModetaNiu**

### CODE REVIEW

**Review by windsonsea — COMMENTED** [review]
navigation.yaml 也要同步修改

**Comment by ModetaNiu** [comment]
> navigation.yaml 也要同步修改

**Comment by ModetaNiu** [comment]
> navigation.yaml 也要同步修改

File: docs/zh/docs/middleware/minio/user-guide/insight-log-config.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+hide:
+  - toc
+---
```
Comment (windsonsea) [inline docs/zh/docs/middleware/minio/user-guide/insight-log-config.md:4]:
```suggestion
```
页面变长以后，右侧的 toc 需要暴露出来

**Review by windsonsea — COMMENTED** [review]
See preview:
https://deploy-preview-4792--daocloud-docs.netlify.app/middleware/minio/user-guide/insight-log-config

**Review by windsonsea — APPROVED** [review]

**Pull request merged**